lib/resolveConfig.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const createStylelint = require('./createStylelint');
+const path = require('path');
+
+/**
+ * Resolves the effective configuation for a given file. Resolves to `null` if
+ * no config is found.
+ * @param {string} filePath The path to the file to get the config for.
```

Comment:
[suggestion] How about inserting `-` to distinguish between a parameter name and description?

```suggestion
 * @param {string} filePath - The path to the file to get the config for.
```